src/frontend/planner_test/tests/testdata/insert.yaml

```diff
@@ -32,32 +32,32 @@
   sql: |
```

Comment:
Are there any e2e tests for this feature?

Reply:
added